src/CoDa.jl

```diff
@@ -29,6 +31,13 @@ export
   norm, dot, ⋅,
   distance,
 
+  # covariances
+  designmatrix,
+  variationmatrix,
```

Comment:
Sorry for renaming requests again, but the picture is becoming clearer now. Can we rename this one to `variation` without the matrix suffix? I think it is also implicit.

Reply:
Solved in commit 36ccc55f38c7eadce190fb1f9eb9b0c780a5ca47.